drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -54,8 +54,10 @@ local function emit_light_status_events(light_device, light)
   if light_device ~= nil then
     if light.status then
       if light.status == "connected" then
+        light_device.log.info_with({hub_logs=true}, "Light status event, marking device online")
```

Comment:
Do we want this logging elevated to hub logs?

Reply:
There are surprisingly few logs related to when these apis are called, so I added it to detect a situation in logs where the hubs Device watch state is not updated (no logging at all) but the api was called.